src/main/scala/org/ergoplatform/network/ErgoNodeViewSynchronizer.scala

```diff
@@ -295,7 +295,7 @@ class ErgoNodeViewSynchronizer(networkControllerRef: ActorRef,
         log.debug(s"$remote has equal header-chain")
     }
 
-    if ((oldStatus != status) || syncTracker.isOutdated(remote) || status == Older || status == Fork) {
+    if ((oldStatus != status) || syncTracker.notSyncedOrOutdated(remote) || status == Older || status == Fork) {
```

Comment:
`isOutdated` returned `true` in these cases (missing, notSynced, outdated), so I renamed the method and changed the implementation so it is more obvious.